client/ayon_houdini/plugins/publish/collect_componentbuilder_lop.py

```diff
@@ -30,19 +30,22 @@ def process(self, instance):
 
         node = hou.node(instance.data["instance_node"])
 
+        try:
+            node.cook(force=True)  # required to clear existing errors
+        except hou.Error as exc:
+            errors = node.errors()
+            if errors:
+                errors = "\n\n - ".join(errors)
+                raise PublishError(
+                    f"Failed to cook node: '{node.path()}'. Please fix it to proceed.",
+                    detail=f"{exc} \n\n - {errors}"
+                )
```

Comment:
```suggestion
                ) from exc
```
How does this look in UI if we raise the error from the original exception?

Reply:
I can't see any difference. 
What should I expect? 
![image](https://github.com/user-attachments/assets/e3c944cd-9f78-418e-878b-00f77e8c7bdf)

Reply:
See the difference of the stack trace in the Houdini Console - it includes information of the original raised error instead of only the one we raised. If it doesn't influence the UI let's add it so that at least the stack trace provides more info on what error actually caused it.

Reply:
printing the error in log is better than using the left side of the publisher window.

Reply:
https://github.com/ynput/ayon-houdini/pull/193/commits/6513a86567e88c9cd983f9c921c6803b960c34ad